### PR TITLE
Start working on documentation

### PR DESCRIPTION
**Pull request by DasMatus**
Work on documentation of this crate, since it really requires one.

### CODE REVIEW

File: src/edupage.rs [hunk 1]
```diff
@@ -41,14 +41,15 @@ impl Default for Edupage {
 }
 
 impl Edupage {
+    /// Builds the client
```
Comment (ivanhrabcak) [inline src/edupage.rs:44]: I think we do not need comments on methods that are self-describing by their name

File: src/edupage.rs [hunk 2]
```diff
@@ -60,7 +61,7 @@ impl Edupage {
             subdomain: None,
         }
     }
-
+    /// Requests the data.
```
Comment (ivanhrabcak) [inline src/edupage.rs:64]: Same here

File: src/traits/dbi.rs [hunk 3]
```diff
@@ -1,8 +1,6 @@
-use crate::{
-    edupage::EdupageError, edupage::Edupage, types::dbi::*, types::person::*,
-};
-
+use crate::{edupage::Edupage, edupage::EdupageError, types::dbi::*, types::person::*};
 
+/// Edupage's DB
```
Comment (ivanhrabcak) [inline src/traits/dbi.rs:3]: Same here.

**Review by ivanhrabcak — COMMENTED** [review]
I appreciate your effort and interest!

File: src/edupage.rs (hunk 1 above)
Comment (DasMatus) [reply src/edupage.rs:44]: Alright. Will remove it.

**Comment by DasMatus** [comment]
Removed every redundant comment, should be ready

**Pull request merged**